alerting/src/main/kotlin/org/opensearch/alerting/settings/AlertingSettings.kt

```diff
@@ -13,13 +13,27 @@ import java.util.concurrent.TimeUnit
 /**
  * settings specific to [AlertingPlugin]. These settings include things like history index max age, request timeout, etc...
  */
+private val log = org.apache.logging.log4j.LogManager.getLogger(AlertingSettings::class.java)
```

Comment:
Once you're done with the development comments, this logger can likely be removed.